README.md

```diff
@@ -104,12 +105,43 @@ evaluator.evaluate()
 risk = evaluator.risk()
 ```
 
+## Configuring logging
+
+`Anonymeter` uses the standard Python logger namespaced under `anonymeter`.
+It inherits the logging configuration from the root logger,
+but you can override it to suit your needs.
```

Comment:
This might be too technical for non expert user. Can you try to add some simpler explanation for the layman?